objc/src/main/java/org/robovm/objc/ObjCObject.java

```diff
@@ -455,25 +507,25 @@ static void release(@Pointer long self, @Pointer long sel) {
             }
             ObjCRuntime.void_objc_msgSendSuper(new Super(self, NS_OBJECT_CLASS).getHandle(), sel);
```

Comment:
I tried that but this either ends in a stack overflow error or everything crashes immediately.
I tried the following:
```Java
ObjCRuntime.void_objc_msgSendSuper(new Super(self, ObjCRuntime.class_getSuperclass(ObjCRuntime.object_getClass(self))).getHandle(), sel);

ObjCRuntime.void_objc_msgSendSuper(ObjCRuntime.class_getSuperclass(ObjCRuntime.object_getClass(self)), sel);
```
Only this works:
```Java
ObjCRuntime.void_objc_msgSendSuper(new Super(self, NS_OBJECT_CLASS).getHandle(), sel);
```